drivers/RF24/RF24.cpp

```diff
@@ -423,13 +423,28 @@ LOCAL void RF24_irqHandler(void)
 		// 1.Read payload, 2.Clear RX_DR IRQ, 3.Read FIFO_status, 4.Repeat when more data available.
 		// Datasheet (ch. 8.5) states, that the nRF de-asserts IRQ after reading STATUS.
 
+#if defined(__linux__)
```

Comment:
Is there a reason we can't do the same for non-linux nodes?

Reply:
There is a small chance of losing packages, but since Linux is not a real-time system, we'll need a workaround